Core/LocalAdmin.cs

```diff
@@ -42,7 +43,7 @@ public sealed class LocalAdmin
 
         public void Start(string[] args)
         {
-            Console.Title = $"LocalAdmin v. {VersionString}";
+            Console.Title = $"LocalAdmin {VersionString}";
```

Comment:
This change doesn't seem relevant to this PR.

Reply:
Kinda it is, I was cleaning up LA output, I did same thing in other places.